reconcile/openshift_resources_base.py

```diff
@@ -275,6 +276,54 @@ def json_to_dict(input):
     return data
 
 
+def urlescape(
+    string: str,
+    quote_plus: bool = False,
+    safe: str = "/",
```

Comment:
@geoberle, as I am adding simple wrappers around the standard library functions, I decided to keep support for this - just in case someone, some future user, would like to have this functionality supported.

However, this might not be as useful as you are pointing out.

Reply:
Done.